### ISSUE TITLE
GLKVIEW ios support in Appium

### ISSUE THREAD

**Issue by chanugithub** [appium/appium#9712]
## The problem

@mykola-mokhnach @imurchie @jlipps  I am unable to get the accessibility id's or any other element id's (in inspector and page source)for GLKView which used heavily in our ios application. does appium ios driver support this? or near future is there any possibility for supporting this feature?

## Environment

* Appium 1.7.1
*Java client 5.0.4
* Selenium 3.6.0
iOS 11.1/xcode 9.1

**Comment by imurchie** [comment]
As of now nothing has been done to support GLKViews. What is reported in the source?

**Comment by chanugithub** [comment]
No info is displayed in the page source, actually trying to automate augmented reality application where user scan image and based on image, a video would be displayed. so trying to get elements  for that video which is using GLKView but nothing is displayed on page source regarding the video

**Comment by imurchie** [comment]
Unfortunately if nothing is reported in the page source then Apple's XCTest framework does not have access to it, and it will not be automatable.
